Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/04-Network-Hub/dev-setup.tf

```diff
@@ -28,14 +28,14 @@ resource "azurerm_subnet_network_security_group_association" "subnet" {
 
 # Linux Server VM
 
-module "create_linuxsserver" {
-  source = "./modules/compute-linux"
+module "create_WindowsClient" {
+  source = "./modules/compute-win"
```

Comment:
planning on deleting `./modules/compute-linux` then as well?

Reply:
yes, we will, as of now it's a module sitting in the code but not getting executed.
Lets wait for a week or two.

Reply:
Curious, why wait?  Should it be removed with its last usage?

Reply:
will remove after discussing with team, if in case it is needed for anything else.

Reply:
okay. will hold up until the conversation has been had.  Thanks.

Reply:
removing the Linux module from the code.